### ISSUE TITLE
Interne und externe Weiterleitung zusammenfassen

### ISSUE THREAD

**Issue by minstyle** [contao/contao#1983]
Bei der internen Weiterelitung einer Seite wäre es schön, wenn man als Weiterleitungsseite auch auf einzelne Nachrichten-Artikel verlinken könnte.

![image](https://user-images.githubusercontent.com/2649562/88385450-a2f21880-cdae-11ea-8ec9-8ffe2907249c.png)

**Comment by aschempp** [comment]
das kannst du doch über die externe Weiterleitung?

**Comment by minstyle** [comment]
> das kannst du doch über die externe Weiterleitung?

Ja, das stimmt. Wenn sich allerdings mal der Nachrichtenalias ändern sollte, dann erzeugen alle externen Links einen 404-Fehler.

**Comment by bytehead** [comment]
Hab mich auch schon gefragt, warum hier der Newspicker nicht auftaucht?

**Comment by fritzmg** [comment]
> Ja, das stimmt. Wenn sich allerdings mal der Nachrichtenalias ändern sollte, dann erzeugen alle externen Links einen 404-Fehler.

If you use the link wizard, you will be able to select all sorts of resources. This will then insert an appropriate insert tag. The link will then be always correct, even if you change the news alias.

**Comment by minstyle** [comment]
> > Ja, das stimmt. Wenn sich allerdings mal der Nachrichtenalias ändern sollte, dann erzeugen alle externen Links einen 404-Fehler.
> 
> If you use the link wizard, you will be able to select all sorts of resources. This will then insert an appropriate insert tag. The link will then be always correct, even if you change the news alias.

Geniale Lösung. Ich wusste garnicht das das geht.

Hier meine Lösung:

![image](https://user-images.githubusercontent.com/2649562/88393670-54e51100-cdbe-11ea-976f-05aa883bb1ec.png)

Trotzdem wäre es schön, wenn es an dieser Stelle auch einen Newspicker geben würde, um die Pflege für Redakteure zu erleichtern.

**Comment by leofeyer** [comment]
Vielleicht sollten wir die Unterscheidung zwischen interner und externer Weiterleitung aufheben und einfach nur noch eine Seite "Weiterleitung" haben, die wie die jetzige externe Weiterleitung funktioniert?

**Comment by contaoacademy** [comment]
Wichtig ist, dass die Funktion der "automatischen Weiterleitung auf die 1. Unterseite" weiterhin möglich ist. Denn dafür nutze ich meist die interne Weiterleitung. Das passiert bei der "externen Weiterleitung" meiner Meinung ja nicht.
